src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -16,23 +16,29 @@ internal sealed class TupleErrorFieldSymbol : SynthesizedFieldSymbolBase
         private readonly TypeSymbol _type;
 
         /// <summary>
-        /// If this field represents a tuple element (including the name match), 
-        /// id is an index of the element (zero-based).
+        /// If this field represents a tuple element with index X
+        ///  2X      if this field represents Default-named element
+        ///  2X + 1  if this field represents Friendly-named element
         /// Otherwise, (-1 - [index in members array]);
```

Comment:
_isImplicitlyDeclared can be a negative number internally. Every field that is not an element (typically Rest) gets a negative Id. It is convenient when we do "Equals" since all ids are distinct.

Externally the API just reports -1. 
The comment on the field is correct. 
The comment on th eproperty shoudl be changed.

---
In reply to: [78255942](https://github.com/dotnet/roslyn/pull/13713#discussion_r78255942) [](ancestors = 78255942)